### PR TITLE
first task completed

### PR DESCRIPTION
**Pull request by MyrzabekDarkhan**

### CODE REVIEW

File: EpamExer/src/main/java/model/Car.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package model;
+
+public class Car {
+    public int id;
+    public String brand;
+    public String model;
+    public String color;
```
Comment (gavrillina) [inline EpamExer/src/main/java/model/Car.java]: why public?

File: EpamExer/src/main/java/model/Car.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    public int price;
+    public int year;
+    public String registerNumber;
+
+
+    public Car(int id, String brand, String model, String color, int price, String registerNumber, int year) {
+        this.id = id;
+        this.brand = brand;
```
Comment (gavrillina) [inline EpamExer/src/main/java/model/Car.java:15]: add one more construction

File: EpamExer/src/main/java/model/Car.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        this.year = year;
+    }
+
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
```
Comment (gavrillina) [inline EpamExer/src/main/java/model/Car.java:26]: do we really need set id?

Comment (MyrzabekDarkhan) [reply EpamExer/src/main/java/model/Car.java:26]: sorry will delete it

File: EpamExer/src/main/java/model/Car.java (hunk 1 above)
Comment (MyrzabekDarkhan) [reply EpamExer/src/main/java/model/Car.java]: change it  to private

**Pull request merged**